src/data-dictionary/events/NrAiIssue/closeCause.md

```diff
@@ -0,0 +1,42 @@
+---
+name: closeCause
+type: attribute
+events:
+  - NrAiIssue
+---
+
+If applicable, what caused the incident to close.
```

Comment:
we tend to avoid having very long attribute definitions like this. I recommend we create a doc for this info and not have it in the attribute.

Reply:
OK, I added like that because I saw it in other attributes. I've moved the info to the [Incident event attributes](https://docs.newrelic.com/docs/alerts-applied-intelligence/new-relic-alerts/advanced-alerts/understand-technical-concepts/violation-event-attributes/)